subprojects/core-api/src/main/java/org/gradle/api/artifacts/transform/ArtifactTransformSpec.java

```diff
@@ -16,18 +16,16 @@
 
 package org.gradle.api.artifacts.transform;
 
-import org.gradle.api.Action;
 import org.gradle.api.Incubating;
 import org.gradle.api.attributes.AttributeContainer;
 
 /**
- * Defines an artifact transformation.
+ * Base configuration for artifact transform registrations.
  *
- * @param <T> The transform specific configuration type.
- * @since 5.2
+ * @since 5.3
```

Comment:
Should this be changed?

Reply:
I changed since the type was essentially new.

Though I think you are right that it should stay at `5.2`. I changed it back.